.github/workflows/github-actions-bazel-build.yml

```diff
@@ -0,0 +1,51 @@
+name: bazel-build
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  Build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Create Cache Timestamp
+        id: cache_timestamp
+        uses: nanzm/get-time-action@v2.0
+        with:
+          format: 'YYYY-MM-DD-HH-mm-ss'
+
+      - name: Mount bazel cache
+        uses: actions/cache@v4
+        with:
+          path: "~/.cache/bazel"
+          key: bazelcache_${{ steps.cache_timestamp.outputs.time }}
+          restore-keys: bazelcache_
+
+      - name: Build
+        run: |
+          # There are no cc_test()s yet, so just build for now.
+          # For now, just run, but don't fail the whole CI, as bazel build is
+          # experimental at this stage.
+          bazel build --keep_going --show_timestamps --curses=no ... || /bin/true
```

Comment:
It would be better to use bazelisk than rely on the native version of bazel.  Why `--keep_going` ?

Reply:
Since we have a `.bazelversion` bazel will bail out anyway if the version doesn't match. So we'll have the correct version of baze.
(I also don't like bazelisk, as it essentially downloads a new version of bazel if it needs to. I want to be in control of that).

`--keep_going` so that if there is an error (which we might expect while working on the bazel build), we don't stop at the first issue but see if there are other things lurking.

Reply:
Its ok for now though in the CI I'd rather not have to manage the bazel version.  That seems more like a local concern for an expert user.

Reply:
We do not have to manage the bazel version. It comes with the github CI, they have all the versions available.